natgeo/mortar
# [SYTO-754] Updates to aspect ratio of parallax frame container per design feedback

This addresses feedback on https://jira.natgeo.com/browse/SYTO-754

To test:

- Pull SYTO-754-3
- Run npm test to verify tests pass
- Run gulp serve
- Navigate to http://localhost:3000/components/podpromo/index.html
- Verify layout and parallax behavior
- Navigate to http://localhost:3000/contentPackages/contentPackage/
- Check layout of bottom two "extra-small" cards at various breakpoints

**Note** There's a known bug where parallax gets a little jumpy when the image and viewport are roughly the same height. That will be addressed in a future story
